src/Software.php

```diff
@@ -981,16 +978,22 @@ public function merge($item, $html = true)
         $ID = $this->getField('id');
 
         if ($html) {
-            echo "<div class='center'>";
-            echo "<table class='tab_cadrehov'><tr><th>" . __('Merging') . "</th></tr>";
-            echo "<tr class='tab_bg_2'><td>";
-            Html::createProgressBar(__('Work in progress...'));
-            echo "</td></tr></table></div>\n";
+            $twig_params = [
+                'merge_msg' => __('Merging'),
+                'progress' => Html::createProgressBar(__('Work in progress...'), ['display' => false])
```

Comment:
Maybe we can use `Html::progressBar()` directly here. `Html::createProgressBar()` is used only twice in our code and is not really usefull (it just forces a `create => true` option).